src/plugins/intel_npu/src/compiler_adapter/include/graph_transformations.hpp

```diff
@@ -15,7 +15,7 @@
  * @brief Contain all required transformation on OpenVINO model in case for external compiler usage and
  *  providing forward compatibility (OV model with opset N+M, external compiler with opset N)
  */
-namespace intel_npu::driverCompilerAdapter {
+namespace intel_npu::driver_compiler_utils {
```

Comment:
Should we rename this to ir_serializer.hpp?

Reply:
done